voter/management/commands/voter_fetch_snapshot.py

```diff
@@ -14,10 +17,6 @@
 s3client = boto3.client('s3')
 s3client.meta.events.register('choose-signer.s3.*', botocore.handlers.disable_signing)
 
-
-NCVOTER_ZIP_URL_BASE = "https://s3.amazonaws.com/dl.ncsbe.gov/data/Snapshots/"
```

Comment:
This URL is different than the one in voter_fetch.py.

* voter_fetch: http://dl.ncsbe.gov.s3.amazonaws.com/data/ncvoter_Statewide.zip
* voter_fetch_snapshot: https://s3.amazonaws.com/dl.ncsbe.gov/data/Snapshots/

At a minimum, we need to remove the trailing `/ncvoter` from NCVOTER_ZIP_URL_BASE and add `/Snapshots/`

Reply:
The beginning part of the URL is also different, but maybe that part doesn't matter (they probably point to the same thing?)

Reply:
Those two URLs point to different things. The "historic" snapshots are in https://s3.amazonaws.com/dl.ncsbe.gov/data/Snapshots/. The live (aka current) voter registration file is at http://dl.ncsbe.gov.s3.amazonaws.com/data/ncvoter_Statewide.zip

Reply:
@rtburg yes! my second comment refers to the fact that http://dl.ncsbe.gov.s3.amazonaws.com/data/ncvoter_Statewide.zip and https://s3.amazonaws.com/dl.ncsbe.gov/data/ncvoter_Statewide.zip are the same file (even though the beginning of the URL looks different).

Reply:
Roger that. Carry on.